metricbeat/module/apache/status/data.go

```diff
@@ -5,17 +5,49 @@ import (
 	"io"
 	"regexp"
 	"strconv"
+	"strings"
 
 	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
 )
 
 // Map body to MapStr
-func eventMapping(body io.ReadCloser) common.MapStr {
+func eventMapping(body io.ReadCloser, hostname string, metricset string) common.MapStr {
 
 	var (
-		totalAccess int
-		totalKB     int
-		uptime      int
+		totalAccess            int
+		totalKB                int
+		uptime                 int
+		cpu_load               float64
+		cpu_user               float64
+		cpu_system             float64
+		cpu_children_user      float64
+		cpu_children_system    float64
+		req_per_sec            float64
+		bytes_per_sec          float64
```

Comment:
I recommend using the Golang naming standard like `BytesPerSec` in case it not conflicts with any auto-name-detection of fields.